README.adoc

```diff
@@ -187,21 +186,20 @@ When injecting, note that, we can provide a specific type - `String` in this cas
 If we have a need for a different `PulsarTemplate` with a different type in the same application, then we can add another injection with that type.
 `PulsarTemplate` currently requires the application to set a default topic.
 Once the topic is set, then we can call the various `send` methods on it.
-In this example, we are calling the very basic `send` method that calls the synchronous send from the Pulsar producer internally that returns the `MessageId`.
+In this example, we are calling the very basic synchronous `send` method that is internally forwarded to the Pulsar producer that returns the `MessageId`.
```

Comment:
I changes this one because the code is not really calling the synchronous send from the Pulsar producer